### PR TITLE
Add 'pending_deployments' endpoint support (#2421)

### PR DESCRIPTION
**Pull request by DavidGamba**

### CODE REVIEW

File: github/actions_workflow_runs.go
Comment (DavidGamba) [inline github/actions_workflow_runs.go:99]: I am passing integers here directly, I wonder if this needs to be wired to some other type.

**Comment by DavidGamba** [comment]
CLA signed.

File: github/actions_workflow_runs.go
Comment (gmlewis) [reply github/actions_workflow_runs.go:99]: Yes, IDs are typically `int64`, so please change this to `[]int64` and remove the leading asterisk.

Comment (DavidGamba) [reply github/actions_workflow_runs.go]: Thanks! fixed.

File: github/actions_workflow_runs.go
Comment (gmlewis) [inline github/actions_workflow_runs.go:101]: Since these two last fields are required, let's please change them to `string` (instead of `*string`).

File: github/actions_workflow_runs.go
Comment (gmlewis) [inline github/actions_workflow_runs.go:97]:
```suggestion
// PendingDeploymentsRequest specifies body parameters to PendingDeployments.
```

File: github/actions_workflow_runs.go
Comment (gmlewis) [inline github/actions_workflow_runs.go:100]: Let's please add a comment to this field:

```suggestion
	// State can be one of: "approved", "rejected".
	State          string `json:"state"`
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Just a few minor tweaks, please @DavidGamba.

**Comment by DavidGamba** [comment]
@gmlewis PTAL

**Review by gmlewis — APPROVED** [review]
Thank you, @DavidGamba !
LGTM.

Awaiting second LGTM from any other contributor to this repo before merging.

**Review by raynigon — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @raynigon !
Merging.

**Pull request merged**
